addons/website/static/src/snippets/s_text_cover/000.scss

```diff
@@ -0,0 +1,19 @@
+.s_text_cover {
+    // Needed to be able to stretch the inner container so that
+    // the snippet works with the 50% and 100% height
+    &.o_half_screen_height, &.o_full_screen_height {
+        > .container, > .container-fluid {
+            &, > .row {
+                min-height: inherit;
+            }
+        }
+    }
+
+    .row {
+        align-items: initial !important;
+
+        & > div {
+            padding: 10rem 3rem;
+        }
```

Comment:
- `> div` instead of `& > div`
- Is this actually needed? Can't you use spacing utilities in the XML directly? That would reduce the number of useless rules + allow us to change our minds on the value of the spacing later without having to deprecate/maintain an old snippet to keep those values.

Reply:
I added the padding via the SCSS because the inline one is reset when switching from grid to cols

Reply:
We don't need to counter that (switching from grid to cols is not really an useful usecase and you can always restore the paddings by yourself if you do that). And if we change our mind saying that we should counter that: it should be generic to all snippets, not a snippet by snippet solution :+1:

Reply:
All right, makes sense, thanks